contracts/UFragmentsPolicy.sol

```diff
@@ -182,6 +182,31 @@ contract UFragmentsPolicy is Ownable {
         rebaseLag = rebaseLag_;
     }
 
+    /**
+     * @notice Sets the hyper-parameters which control the timing and frequency of
+     *         rebase operations.
+     *         a) the minimum time period that must elapse between rebase cycles.
+     *         b) the rebase window offset parameter.
+     *         c) the rebase window length parameter.
+     * @param minRebaseTimeIntervalSec_ More than this much time must pass between rebase
+     *        rebase operations, in seconds.
+     * @param rebaseWindowOffsetSec_ The number of seconds from midnight when
+     *        the rebase window begins.
+     * @param rebaseWindowLengthSec_ The size of the rebase window in seconds.
+     */
+    function setRebaseTimingParameters(uint256 minRebaseTimeIntervalSec_,
+        uint256 rebaseWindowOffsetSec_, uint256 rebaseWindowLengthSec_)
+        external
+        onlyOwner
+    {
+        require(minRebaseTimeIntervalSec_ > 0);
```

Comment:
This is not needed. Because in line 97 require(lastRebaseTimestampSec.add(minRebaseTimeIntervalSec) < now);
< is used so there is not a categorical difference between 0 and 1. 0 means 1 second and 1 means 2 seconds.

Reply:
👍

Reply:
Added this back